src/solver/misc/options.cpp

```diff
@@ -84,17 +84,28 @@ std::unique_ptr<GetOpt::Parser> CreateParser(Settings& settings,
 
     // add option for ortools use
     // --use-ortools
-    parser->addFlag(
-      options.ortoolsUsed, ' ', "use-ortools", "Use ortools library to launch solver");
+    parser->addFlag(options.optOptions.ortoolsUsed,
+                    ' ',
+                    "use-ortools",
+                    "Use ortools library to launch solver");
 
     //--ortools-solver
-    parser->add(options.ortoolsSolver,
+    parser->add(options.optOptions.ortoolsSolver,
                 ' ',
                 "ortools-solver",
                 "Ortools solver used for simulation (only available with use-ortools "
                 "option)\nAvailable solver list : "
                   + availableOrToolsSolversString());
 
+    //--xpress-parameters
+    parser->add(
+      options.optOptions.solverParameters,
+      ' ',
+      "solver-parameters",
+      "Set xpress solver specific parameters. The specified string must be wrapped into quotes: "
```

Comment:
Set solver-specific parameters, for instance --solver-parameters="THREADS 1 PRESOLVE 1" for XPRESS or --solver-parameters="parallel/maxnthreads 1, lp/presolving TRUE" for SCIP. Syntax is solver-dependent, and only supported for SCIP & XPRESS.